src/data/markdown/docs/01 guides/03 Results visualization/Cloud.md

```diff
@@ -0,0 +1,77 @@
+---
+title: 'Cloud'
+excerpt: ''
+---
+Besides [running cloud tests](/cloud/creating-and-running-a-test/cloud-tests-from-the-cli), you can also stream your test results in real-time to the [k6 Cloud](/cloud).
+
+The k6 Cloud will pre-process your data, and you can visualize and analyze the results on the web app.
+
+## Instructions
+
+**1 (Optional) - Log in to the k6 Cloud**
+
+Assuming you have installed k6, the first step is to log in to k6 Cloud:
+
+<div class="code-group" data-props='{"labels": ["Log in to k6 Cloud", "Log in with the API token"]}'>
```

Comment:
I would split these into two code groups, or at least show both at the same time, rather than using tabs. I also think using the API token might be the option we want to highlight, as that is how most CI users will (or definitely **should**) use it.

Reply:
I wanted to show "simplicity" this is why I decided with this idea.

I also think it might not be the best solution. But let me get more feedback here.

https://trello.com/c/wd71oiEj/204-upload-results

Reply:
I agree with @simskij , that the token is more likely to be the actual way someone will use it in a CI setup.